pkg/controller/catalog/storage/tpr_storage_binding.go

```diff
@@ -0,0 +1,84 @@
+package storage
+
+import (
+	"errors"
+	"log"
+
+	"github.com/kubernetes-incubator/service-catalog/pkg/apis/servicecatalog"
+	"github.com/kubernetes-incubator/service-catalog/pkg/controller/catalog/util"
+	"github.com/kubernetes-incubator/service-catalog/pkg/controller/catalog/watch"
+	"k8s.io/client-go/1.5/pkg/api/v1"
+	"k8s.io/client-go/1.5/pkg/runtime"
+)
+
+type tprStorageBinding struct {
+	watcher *watch.Watcher
+	ns      string
+}
+
+func newTPRStorageBinding(watcher *watch.Watcher, ns string) *tprStorageBinding {
+	return &tprStorageBinding{watcher: watcher, ns: ns}
+}
+
+func (t *tprStorageBinding) Update(in *servicecatalog.Binding) (*servicecatalog.Binding, error) {
+	in.Kind = watch.ServiceBindingKind
+	in.APIVersion = watch.FullAPIVersion
+	tprObj, err := util.SCObjectToTPRObject(in)
+	if err != nil {
+		log.Printf("Failed to convert object %#v : %v", in, err)
+		return nil, err
+	}
+	tprObj.SetName(in.Name)
+	log.Printf("Updating Binding %s in k8s:\n%v\n", in.Name, tprObj)
+	_, err = t.watcher.GetResourceClient(watch.ServiceBinding, "default").Update(tprObj)
+	// krancour: Ideally the binding we return is a translation of the updated 3pr
+	// as read back from k8s. It doesn't seem worth going through the trouble
+	// right now since 3pr storage will be removed soon. This will at least work
+	// well enough in the meantime.
+	return in, err
+}
+
+// ListServiceBindings returns all the bindings
```

Comment:
yea I was kind of surprised that golint didn't complain about this too. I think there were missing comments on other exported funcs in this file  - was this file excluded from the lint checks?